connect-example/src/main/java/com/stripe/android/connect/example/ui/common/BasicExampleComponentActivity.kt

```diff
@@ -125,13 +108,6 @@ abstract class BasicExampleComponentActivity : FragmentActivity() {
                     openSettings = openSettings,
                     reload = viewModel::reload,
                 ) { embeddedComponentManager ->
-                    val context = LocalContext.current
-                    LaunchedEffect(context) {
-                        val appearanceInfo = settingsService.getAppearanceId()
-                            ?.let { AppearanceInfo.getAppearance(it, context).appearance }
-                            ?: return@LaunchedEffect
-                        embeddedComponentManager.update(appearanceInfo)
```

Comment:
👀 Binding the app appearance setting to the manager is now done in `EmbeddedComponentLoaderViewModel`